components/Dialogs/tests/unit/DialogsNoopTest.m

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialDialogs.h"
+
+@interface DialogsNoopTest : XCTestCase
+
+@end
+
+@implementation DialogsNoopTest
+
+- (void)testSimpleBuild {
+  MDCAlertController *controller = [[MDCAlertController alloc] init];
```

Comment:
init should actually be NS_UNAVAILABLE.
Please replace with:
MDCAlertController *controller = [MDCAlertController alertControllerWithTitle:@"Foo" message:"@"Bar"];

Reply:
You want me to change the implementation of MDCAlertController?

Reply:
We should match UIAlertController's behavior.

[[UIAlertController alloc] init] will create an object.  It will throw an exception when you present it unless you fill in the details, but that is unrelated.

+1 LGTM.